app/views/hiring_staff/vacancies/edit_vacancy_sections/_edit_application_details.html.haml

```diff
@@ -14,7 +14,11 @@
     %dd.app-check-your-answers__answer.first-question
       =mail_to 'Job contact email', @vacancy.contact_email, class: 'govuk-link', 'aria-label': t('jobs.aria_labels.contact_email_link', email: @vacancy.contact_email)
 
-  - organisation = @vacancy.parent_organisation.is_a?(SchoolGroup) ? 'trust_visits' : 'school_visits'
```

Comment:
You don't mention removing this info in the PR. Was it accidental?

Reply:
Don't see a reference to this in Jira

Reply:
It was useless code

Reply:
Oh I see it's just an assignment, not a thing on the page